regen/feature.pl

```diff
@@ -37,6 +37,7 @@ BEGIN
     signatures      => 'signatures',
     isa             => 'isa',
     indirect        => 'indirect',
+    trim            => 'trim',
```

Comment:
This file needs a docs section for the 'trim' feature, which will then apply to feature.pm once regenerated.

Reply:
I'm 90% sure this is a @LeoNerd thing? I didn't do this piece.